python/requirements/data_processing/requirements.txt

```diff
@@ -0,0 +1,2 @@
+dask[complete]==2021.03.0; python_version < '3.7'
+dask[complete]==2021.06.1; python_version >= '3.7'
```

Comment:
Hmm shouldn't dask already be installed for the other dask on ray test? Could we grab that requirements file instead to avoid duplication?

Reply:
The other requirements file (that includes Dask) includes all Tune dependencies, which we don't need, and the datasets/workflows job is a different job than those Dask-on-Ray tests (which run in the same job as Tune and DistributedSGD).